XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/package.json

```diff
@@ -10,7 +10,7 @@
     "mixed",
     "reality"
   ],
-  "version": "0.2.6",
+  "version": "0.2.7",
```

Comment:
Let's wait to update this until right before we do a release

Reply:
Fixed